pkg/networkmanager/interfaces.js

```diff
@@ -3670,7 +3670,8 @@ PageNetworkBondSettings.prototype = {
         function change_mode() {
             options.mode = select_btn_selected(mode_btn);
 
-            primary_btn.parents("tr").toggle(options.mode == "active-backup");
+            primary_btn.toggle(options.mode == "active-backup");
+            primary_btn.prev().toggle(options.mode == "active-backup");
```

Comment:
Just to make sure I understand this: usually `element.prev()` would be the corresponding label, and we want to show/hide both the label and the element. Before this was done by showing/hiding all elements in the row, which should amount to the same effective thing?

Reply:
Indeed. With current html I have from garret labels and elements are not nested in some containers element so we need to introduce these two steps instead of the one that was before.